### ISSUE TITLE
A recent update seems to have broken it

### ISSUE THREAD

**Issue by anoided** [shmykelsa/AAAD#161]
The installed applications no longer show up on the screen when trying to open them.
Everything was previously working fine and I am assuming there was some android auto update that broke it.
I updated AAAD, removed application(carstream in this case) reinstalled carstream. 
Unable to select carstream or add it to the launcher in android auto. I tried screen2mirror as well but the result was the same, like android auto no longer recognizes them as proper auto apps.

**Comment by churritin01** [comment]
The only way it works now is using the indiegogo CAR Dongle, at least Fermata works great 👍 ![20211205_232151.jpg](https://user-images.githubusercontent.com/83697330/144787841-14bd3f4f-0ee8-4a42-a779-f942d8132328.jpg)https://user-images.githubusercontent.com/83697330/144787857-c48a039d-8f34-4c3d-a9d7-6e5fc80b41d3.mp4![IMG-20211205-WA0083.jpeg](https://user-images.githubusercontent.com/83697330/144787869-4553494d-6485-4493-b69d-f070e68af879.jpeg)

**Comment by anoided** [comment]
So I played with it some more today. I tried uninstalling android auto or i guess going to the first version. My intent was to then download an older version and install it and not allow it to update since it had previously worked. Removing and reinstalling apps through aaad didnt seem to make any difference as carstream still didnt show up in the launcher menu. So I sort of gave up on the version I was trying and was going to go back another one, it was 6.9 at the moment. I removed android auto with car stream already installed and when I plugged it into the car it updated aa and installed the latest version. This time however car stream was on the launcher.
